counter/counter.coco

```diff
@@ -6,20 +6,20 @@ coco Counter
 // values managed intrinsically by the logic. Only 
 // a single call can access these values at a time.
 state persistent:
-    Value U64
+    value U64
 
-// Init!() deployer is a constructor that leaves
-// the persistent state untouched at the deploy time.
-endpoint deployer Init!():
+// Init!() deployer is a constructor that initializes the persitent state
```

Comment:
update the comment to use `Init` without the exclamation. applies to other endpoints in all modules in the PR

Reply:
Removed "!" marks from all the modules